modules/apps/frontend-js/frontend-js-recharts/npmscripts.config.js

```diff
@@ -13,20 +13,9 @@
  */
 
 module.exports = {
-	'create-jar': false,
-	exports: {
-		recharts: 'recharts',
-		recharts_lib_index: 'recharts/lib/index.js',
+	federation: {
+		bridges: ['recharts'],
+		runLegacyBuild: false,
```

Comment:
Just to check, is the mixing in of this other LPS in here strictly necessary? I have no problem with you having it go along for the ride; I'm just curious.

Reply:
It's a prerequisite. If I don't remove bundler 3 it doesn't build unless I "fix" it. And we don't want to fix bundler 3 up to this point...

Reply:
Farewell, Bundler 3... I hardly knew you... 😢

![farewell](https://user-images.githubusercontent.com/7074/109136427-662d3c00-7758-11eb-8ff9-f804aa8cb3d8.gif)